src/platform/unix/mod.rs

```diff
@@ -0,0 +1,1029 @@
+// Copyright 2015 The Servo Project Developers. See the COPYRIGHT
```

Comment:
Something went wrong with your commit: it introduces both ` src/platform/unix/mod.rs` and `src/platform/unix/linux/mod.rs` (with identical contents) in place of the original `src/platform/linux/mod.rs`...